utils/cpuload/netlink.go

```diff
@@ -0,0 +1,232 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cpuload
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"syscall"
+)
+
+const (
+	GENL_ID_CTRL            = syscall.NLMSG_MIN_TYPE
```

Comment:
Having the same name as kernel consts makes it easier to lookup in kernel sources :(

Changed it to go fmt, but added kernel equivalent as comments.